src/DataValues/DecimalValue.php

```diff
@@ -91,7 +91,12 @@ private function convertToDecimal( $number ) {
 			throw new InvalidArgumentException( '$number must not be NAN or INF.' );
 		}
 
-		$decimal = strval( abs( $number ) );
+		// Enforce the default, see http://php.net/manual/en/ini.core.php#ini.serialize-precision
```

Comment:
Would be nice to document why the strange detour via serialize() is needed.

Reply:
Uh, I'm not sure what would be helpful as a comment here. We could stick to strval, but would need to temporarily disable the locale then. json_encode is an other option. Do you think this is better?

Reply:
Maybe:
```
// As PHP has no other built-in way to format a number with a given number of significant digits,
// use serialize. While strval would also work for this, it is locale-dependent.
```

Reply:
Maybe:
```
// As PHP has no other built-in way to format a number with a given number of significant digits,
// use serialize. While strval would also work for this, it is locale-dependent.
```

Reply:
sprintf would probably be the cleaner solution, as proposed by marius below. If that's not possible, a comment should explain why.